src/__tests__/cms-page/[...slug].test.tsx

```diff
@@ -77,11 +77,11 @@ describe.skip('CMS Page', () => {
       page,
     });
 
-    (
```

Comment:
Related to this code in this file:
```javascript
// next-page-tester doesn't support next v12 yet. PR here: https://github.com/toomuchdesign/next-page-tester/issues/281
```

Read https://github.com/next-page-tester/next-page-tester?tab=readme-ov-file#this-project-is-deprecated and e.g.
```javascript
// next-page-tester is deprecated & incompatible with next v13, README suggests using browser testing instead
```
would be more accurate IMHO.

Reply:
fixed